lib/capistrano/stretcher/tasks/rails.rake

```diff
@@ -13,41 +13,45 @@ namespace :stretcher do
 
   task :bundle do
     on application_builder_roles do
-      within local_build_path do
-        execute :bundle, :install, "--gemfile #{local_build_path}/Gemfile --deployment --path #{local_bundle_path} -j 4 --without development test"
+      with rbenv_version: fetch(:rbenv_version) do
```

Comment:
What's happened without rbenv environment? I'm not sure that we should specify `rbenv_version` variable.

Reply:
If rbenv_version is not specified, version of global was used

Reply:
👍